<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gererTypesTables</class>
 <widget class="QDialog" name="gererTypesTables">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GÃ©rer les types de tables</string>
  </property>
  <widget class="QListWidget" name="listeTypesTables">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>211</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="nouveauTypeTableButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>166</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Nouveau type de table</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Types de tables</string>
   </property>
  </widget>
  <widget class="QWidget" name="typeTableView" native="true">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>79</y>
     <width>291</width>
     <height>211</height>
    </rect>
   </property>
   <widget class="QLabel" name="numeroLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>62</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="libelleLineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>62</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Numero</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>62</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Libelle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="appliquerPushButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>92</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Appliquer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="supprimerButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>92</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Supprimer</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
